src/app/features/collectibles/components/collectible-text.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Text } from '@stacks/ui';
+
+import { useTextInscriptionContentQuery } from '@app/query/bitcoin/ordinals/use-text-ordinal-content.query';
+
+import { CollectibleLayout, CollectibleLayoutProps } from './collectible.layout';
+
+function BottomFadeOut() {
+  return (
+    <Box
+      sx={{
+        position: 'absolute',
+        bottom: '0',
+        height: '30px',
+        width: '100%',
+        backgroundImage: 'linear-gradient(rgba(0,0,0,0), rgba(0,0,0,1))',
+      }}
+    />
+  );
+}
```

Comment:
Would better be implemented as a pseudo element

Reply:
What's the benefit of having it as a pseudo element?

Reply:
https://css-tricks.com/7-practical-uses-for-the-before-and-after-pseudo-elements-in-css/#aa-gradient-overlays